queries.sql

```diff
@@ -64,3 +64,21 @@ SELECT MAX(escape_attempts) FROM animals;
 SELECT MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species;
 
+/* write queries to answer the questions */
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.id = 4;
+
+SELECT animals.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.id = 1;
+
+SELECT animals.name, owners.fulL_name FROM owners LEFT JOIN animals ON owners.id = animals.owner_id;
+
+SELECT species.name, COUNT(*) FROM animals JOIN species ON species.id = animals.species_id GROUP BY species.name;
+
+SELECT animals.name, owners.full_name, species.name FROM animals
+JOIN species ON species.id = animals.species_id
+JOIN owners ON owners.id = animals.owner_id
+WHERE owners.id = 2 AND species.name = 'Digimon';
```

Comment:
Same issue here 🤓 Kindly use `owners.full_name`  column instead of  `owners.id = 2` 😬

Reply:
Updated. Thanks